modules/microshift-proc-deploying-applications-with-generic-devices.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// microshift_configuring/microshift-gdp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-deploying-applications-with-generic-devices_{context}"]
+= Deploying applications that use generic devices
+
+After the Generic Device Plugin (GDP) is configured and enabled in {microshift-short}, you can deploy Kubernetes workloads, such as pods, deployments, or StatefulSets, that request access to the host devices that you have exposed. The device is made available inside the container without requiring the pod to run with elevated privileges.
+
+.Prerequisites
+
+* You installed {product-title}.
+
+* You enabled and configured GDP.
+
+* You installed {oc-first}.
+
+.Procedure
+
+. Define the device request in your `Pod` specification:
++
+[IMPORTANT]
+====
+In your Kubernetes workload manifest, for example, for a pod, deployment, or StatefulSet, add a `resources.limits` entry to the container that needs device access. The resource name must follow the pattern `device.microshift.io/<device_name>`, where `<device_name>` matches the `name` that you specified in your configuration file. The value indicates the number of instances of that device type the container requires.
+====
++
+[source,yaml]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: device-app
+spec:
+  containers:
+  - name: container
+    image: <your_application_image> <1>
+    command: ["/path/to/your/app"] <2>
+    args: ["--device_path=/dev/video0"] <3>
+    resources:
+      limits:
+        device.microshift.io/video: 1 <4>
+    securityContext: <5>
+      allowPrivilegeEscalation: false
+      capabilities:
+        drop: ["ALL"]
+      runAsNonRoot: true
+      seccompProfile:
+        type: "RuntimeDefault"  
+----
+<1> Replace with your container image.
+<2> Replace with the command for your application.
+<3> For example, how your application might use the device.
+<4> A request for one instance of the `video` device.
+<5> Define and configure with the least privilege value to ensure that the container has only required permissions, such as access to the device file, and to restrict other capabilities for the container.
++
+. Deploy the Kubernetes workload by applying the manifest to the {microshift-short} cluster by running the following command:
+
+[source,terminal,subs="+quotes"]
+----
+$ oc apply -f _<your-workload-manifest.yaml>_ <1>
+----
+<1> Replace _<your_workload_manifest>_ with the name of your workload manifest.
+
+.Verification
```

Comment:
```suggestion
.Results
```
our use of Verification is pretty strict that it should be bulleted steps only; Jaromir recently demo'd the .Results header as valid for DITA, so I would go ahead and use it here